apidoc/Titanium/Android/Intent.yml

```diff
@@ -217,23 +316,33 @@ examples:
             });
             intent.addCategory(Ti.Android.CATEGORY_DEFAULT);
 
-  - title: Create an ImageView from an Image Send Intent 
+  - title: Create an ImageView from an Image Send Intent
     example: |
-        This example requires that an intent filter be set up in a
-        custom AndroidManifest.xml. The following intent filter should
-        be added to your app's main activity. You can trigger this 
-        intent filter by long pressing on an image in the Android gallery
+        This example requires that an intent filter be set up in the project's `tiapp.xml` file.
+        After copying the default root activity of your application from the `AndroidManifest.xml`
```

Comment:
"After copying the default root activity of your application from the AndroidManifest.xml file to the Android manifest section of the tiapp.xml file..."

Is this the AndroidManifest.xml in the build folder? I'm not sure I would know how to do this, i.e. exactly what to copy and to where.